examples/md-flexible/src/configuration/MDFlexConfig.h

```diff
@@ -125,12 +125,6 @@ class MDFlexConfig {
    */
   void calcSimulationBox();
 
-  /**
-   * Returns the particles generated based on the provided configuration file.
-   * @return a vector containing the generated particles.
-   */
-  std::vector<ParticleType> getParticles() { return _particles; }
-
```

Comment:
Was there a reason to not use the getter but instead make the vector public?

Reply:
Usually getters should be `const` methods (so this getter was already misleading) that only provide read access. Write access usually should happen via other methods (setters, dedicated functions etc....) . However, here, I need full read and write access to the actual vector. Hence, I made it public because it is treated as such.